gradle.properties

```diff
@@ -14,6 +14,6 @@
 # limitations under the License.
 #
 
-version=3.30.0
+version=3.32.0
```

Comment:
Probably revert this and just change it manually, I think this pr will might merged before the other one.

Reply:
32 is the correct next version because the last release 3.31.0 didn't come with gradle.properties update. The other PR will be 3.33.0